### PR TITLE
Added Integration test template

### PR DESCRIPTION
**Pull request by minakawa-daiki**
I have added a template for future implementation of integration tests (#56) in rust.
To run it, use `cargo test --test integration`.

Screenshot from 2021-06-06 22-36-07

### CODE REVIEW

File: tests/integration.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+use std::env;
+use std::process::{Command, Stdio};
+
+#[test]
+fn main() {
+    let path_result = env::current_dir();
+    let path = match path_result {
+        Ok(path) => path.display().to_string(),
```
Comment (utam0k) [inline tests/integration.rs:8]: Can you use PathBuf?

File: tests/integration.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        Err(_) => panic!("Path is not found"),
+    };
+    let status = Command::new(path + "/target/x86_64-unknown-linux-gnu/debug/youki")
```
Comment (utam0k) [inline tests/integration.rs:11]: If you want to use the pre-built one, why not simply merge the paths with youki?

**Review by utam0k — CHANGES_REQUESTED** [review]

File: tests/integration.rs (hunk 2 above)
Comment (minakawa-daiki) [reply tests/integration.rs:11]: https://github.com/containers/youki/blob/adb43102e022d05c4730fa95e431b30050dec7d8/integration_test.sh#L9

I was just following your direct specification here, but if you want to just merge the youki paths, I'll do that!

**Comment by minakawa-daiki** [comment]
@utam0k How about a fix based on your review?

File: tests/integration.rs (hunk 2 above)
Comment (utam0k) [reply tests/integration.rs]: I think you just need to connect it to the `youki` path. That code is a bit out of date.

**Comment by utam0k** [comment]
@minakawa-daiki Ok, source codes look good. Why don't you add a README? How about stating there that it is a preview implementation and how to run it?

**Comment by minakawa-daiki** [comment]
@utam0k I added it to the README.md

**Review by utam0k — APPROVED** [review]
nice work!

**Review by utam0k — CHANGES_REQUESTED** [review]
sorry, the ci is failed. can you fix it?

**Comment by minakawa-daiki** [comment]
@utam0k 
https://github.com/containers/youki/pull/71#discussion_r646139293
It seems that youki Path pointed out here does not exist and CI has been dropped. I think the solution is to add `"/target/x86_64-unknown-linux-gnu/debug/youki"` to the path before you pointed it out, what do you think?

**Comment by minakawa-daiki** [comment]
@utam0k The CI has been passed, please check.

**Review by utam0k — APPROVED** [review]
It's a good first step.

**Review by utam0k — APPROVED** [review]

**Pull request merged**